### PR TITLE
Added 4 channels, updated Canal Catorce

### PR DESCRIPTION
**Pull request by GerardoHD**

### CODE REVIEW

File: channels/mx.m3u [hunk 1]
```diff
@@ -1,10 +1,14 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="http://i.imgur.com/MeaSmiv.jpg" group-title="",15TV Sabinas
 http://live.gvstream.net:1935/e-tv-video/play.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/amastv/picture?width=200&height=200" group-title="",A+
+https://dai.google.com/linear/hls/event/SJysMl45QMSwjo0TodSk1Q/master.m3u8
```
Comment (freearhey) [inline channels/mx.m3u:5]: Link does not work.

File: channels/mx.m3u [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/x8lwsjM.png" group-title="",ADN 40
 https://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/9Xn7ZEZ.jpg" group-title="",Alcarria TV
 http://217.182.77.27/live/alcarriatv-livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/aztecauno/picture?width=200&height=200" group-title="",Azteca uno
+https://dai.google.com/linear/hls/event/asXufUKSSKisiLSMHR2BXA/master.m3u8
```
Comment (freearhey) [inline channels/mx.m3u:11]: Link does not work.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: channels/mx.m3u (hunk 2 above)
Comment (GerardoHD) [inline channels/mx.m3u:11]: https://pubads.g.doubleclick.net/ssai/event/asXufUKSSKisiLSMHR2BXA/master.m3u8

It's another link that i have, but as i'm using AdGuard DNS, i switched to the ones i send first.

File: channels/mx.m3u (hunk 1 above)
Comment (GerardoHD) [inline channels/mx.m3u:5]: https://pubads.g.doubleclick.net/ssai/event/SJysMl45QMSwjo0TodSk1Q/master.m3u8

Maybe that one works, or they are geo-blocked?

**Review by GerardoHD — COMMENTED** [review]
I put old links that should work, maybe they are geo-blocked, but i don't believe is that

**Comment by GerardoHD** [comment]
Sorry if i didn't do the review process correctly.

File: channels/mx.m3u (hunk 2 above)
Comment (freearhey) [reply channels/mx.m3u:11]: Both broadcasts do not work for me, I only see a black screen.

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/mx.m3u (hunk 2 above)
Comment (GerardoHD) [reply channels/mx.m3u:11]: May these two channels be ignored?
I don't see a better option at this moment, thank you for everything

**Review by freearhey — APPROVED** [review]

**Pull request merged**